### ISSUE TITLE
synthetics-minion should use MINION_PRIVATE_LOCATION_KEY env var

### ISSUE THREAD

**Issue by keegoid-nr** [newrelic/helm-charts#942]
**Bug description**

Commit 04818bcdd2c0f7fa9bbe745e565332be28e7d38a may have inadvertently changed the `MINION_PRIVATE_LOCATION_KEY` environment variable to `PRIVATE_LOCATION_KEY`.

**Version of Helm and Kubernetes**

All versions

**Which chart?**

1.0.52

**What happened?**

CPM version 3.0.65 still requires the env var to be called `MINION_PRIVATE_LOCATION_KEY`.

The following error is returned if not present:

```log
$ kubectl logs synthetics-minion-0 
2022-09-09 23:03:21,737 - Minion will use 1 heavy workers  
2022-09-09 23:03:21,738 - Minion will use 25 lightweight workers  
2022-09-09 23:03:21,741 - ENV Variable 'MINION_PRIVATE_LOCATION_KEY' must be provided
```

And the minion pod exits with an Error state, leading to:

```log
NAME                  READY   STATUS             RESTARTS        AGE
synthetics-minion-0   0/1     CrashLoopBackOff   8 (4m15s ago)   20m
```

**What you expected to happen?**

Need to revert this change in order for the minion to pass the init phase.

**How to reproduce it?**

```sh
helm add repo newrelic https://helm-charts.newrelic.com
helm repo update
helm upgrade --install synthetics-minion -n nr-minion newrelic/synthetics-minion -f values.yaml --create-namespace
```

**Comment by KrisM4c** [comment]
Yeah I'm seeing the exact same. Can see they updated on this commit but looks like the app is still expecting `MINION_PRIVATE_LOCATION_KEY`

https://github.com/newrelic/helm-charts/commit/5365e6bf648bb208240f3c569448f0143c675d2b#diff-1b6a24e5462b223e67ab4a7421e19e57b109a49fd90c91772cf0baabf5306a9bR121

**Comment by vmaillot** [comment]
i confirm as well this issue here, i hacked the sts directly to make it work, and it looks like we got the [pr](https://github.com/newrelic/helm-charts/pull/958) ready! thanks o/

**Comment by keegoid-nr** [comment]
This has been resolved in chart version 1.0.53. Closing.
https://github.com/newrelic/helm-charts/commit/41f45bc71ebd5b7f91a3ba531bf6dd295922adb7
